integrations/adobe-analytics/lib/index.js

```diff
@@ -125,10 +129,13 @@ AdobeAnalytics.global('s')
  * @api public
  */
 
-AdobeAnalytics.prototype.initialize = function() {
+AdobeAnalytics.prototype.initialize = function () {
   var options = this.options;
   var self = this;
 
+
+  options.chromecastMode = true; //Hard coding this to true for this branch, will be switched to a flag/setting if merged into main
```

Comment:
Instead of hard-coding which creates noise for all the tests, can we ask the customer to update their implementation to include this setting using the integrations object?

https://segment.com/docs/connections/spec/common/#integrations

I believe you can do something like

```json
"integrations": {
  "Adobe Analytics": {
    "chromecastMode": true
  }
}

Reply:
discussed offline with Marcus and we will keep this hardcoded for now. We will pin a version of a.js with this code to Fox's Chromecast sources for testing. In the short/medium-term, we plan to roll these changes into the existing Adobe integration and adding the boolean setting to partner portal, at which point we'll update this to reference the setting (instead of hardcoding).